axelor-account/src/main/resources/reports/Invoice.rptdesign

```diff
@@ -5583,7 +5583,7 @@ else if(row["contactPartnerTitleSelect"] == "4"){'Prof.'}</value>
                             </structure>
                         </list-property>
                         <property name="contentType">html</property>
-                        <text-property name="content"><![CDATA[<VALUE-OF>if(params["Locale"].value != 'en') {
+                        <text-property name="content"><![CDATA[<VALUE-OF>if(params["Locale"].value != 'en_US') {
```

Comment:
no you have to extract the language value in the Locale, it has to work even for en_UK for example